README.md

```diff
@@ -500,6 +500,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Honey Patel](https://github.com/honeykpatel)
   - [Hrithik Chauhan](https://github.com/Hrithik5)
   - [Hulya Karakaya](https://github.com/hulyak)
+  - [Hana Shah](https://github.com/hanashah-01)
```

Comment:
Please add your name in line 482, after Hana Aliyah Mufidah.
Let us know if you need any help doing this! 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
Hi, may I know do I have to make a new pull request or just commit the new changes?

Reply:
You can commit the changes and the PR will be updated automatically. 🙂


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Tags:** Reusable